dcore/configuration.py

```diff
@@ -0,0 +1,81 @@
+"""
+Some simple tools for making model configuration nicer.
+"""
+
+
+class Configuration(object):
+    def __init__(self, **kwargs):
+
+        self.__setattr__ = self._proto__setattr__
+        for name, value in kwargs.iteritems():
+            print name, value
```

Comment:
So the code documents the parameters used, right?  If you want to more formally record the parameter state used to run a simulation and dump it such that you can later recreate, then we could think about how to do that.  But printing out the values in the log file doesn't feel like the right way.